Detectors/TRD/reconstruction/macros/checkTrackletCharges.C

```diff
@@ -23,9 +40,9 @@ using namespace o2::trd;
 std::vector<TH1F*> createTrdChargeHists()
 {
   std::vector<TH1F*> hCharges;
-  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 64, 0, 63));
+  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 256, 0, 255));
   hCharges.push_back(new TH1F("Charge1", "Charge window 1;charge;counts", 256, 0, 255));
-  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 256, 0, 255));
+  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 128, 0, 127));
```

Comment:
I think that's incorrect: the ranges should be 256, 0.0, 256.0. Or, actually, I usually prefer 256, -0.5, 255.5 to have the labels in the bin centers.

Reply:
oui oui, they can all potentially be 8 bit :-( However this is just plotting them, not manipulating data, so its more a question of clear easily readable graphs, no? and given that most of the time its going to be 7 7 6 when not using the scaled q values, is it not clearer to h ave 255.5 255.5 127.5 ? giving an area at the right with nothing and clearly showing the spike? Yes I definitely agree about the -0.5 to -???.5

Reply:
@tdietel : could you approve, then we'Ll merge it

Reply:
Restricting it to 7/7/6 bits is fine. Then the ranges should be 
128, -0.5, 127.5
64, -0.5, 63.5
Please update